Gemfile

```diff
@@ -34,14 +34,15 @@ gem 'bcrypt', '~> 3.1.7'
 
 gem 'bootsnap', '>= 1.1.0', require: false
 
-
 group :development, :test do
   # Call 'byebug' anywhere in the code to stop execution and get a debugger console
   gem 'byebug', platforms: [:mri, :mingw, :x64_mingw]
   gem 'pry'
 end
 
 group :test do
+  gem 'faker'
+  gem 'factory_bot_rails'
```

Comment:
This position has changed drastically.  Just make sure that it's under rails_helper gem and you're golden.